BUILD.bazel

```diff
@@ -72,6 +72,17 @@ python_grpc_compile(
     deps = [":node_manager_proto"],
 )
 
+proto_library(
+    name = "job_info_access_proto",
+    srcs = ["src/ray/protobuf/job_info_access.proto"],
```

Comment:
In that file, we can define multiple `XInfoGcsService` services. On the client side, we can only have one `GcsRpcClient` class that contains all methods, because these methods are pretty simple. On the server side, current approach seems fine. But we can think about how to better organize code, especially for those boilerplate code.